src/Model.php

```diff
@@ -1225,14 +1234,45 @@ public function addCondition($field, $operator = null, $value = null)
             */
         }
 
-        $f = is_string($field) ? $this->getField($field) : ($field instanceof Field ? $field : false);
-        if ($f) {
+        if (is_string($field)) {
+            // shorthand for adding conditions on references
+            // use chained reference names separated by "/"
+            if (stripos($field, '/') !== false) {
+                $references = explode('/', $field);
+
+                $field = array_pop($references);
+
+                $model = $this;
+                foreach ($references as $link) {
+                    $model = $model->refLink($link);
+                }
+
+                $args = func_get_args();
+
+                // '#' will apply condition directly on the record count (has # referenced records)
+                // otherwise applying condition on the referenced model field (has referenced records where)
+                if ($field !== '#') {
+                    $model->addCondition(...$args);
+                    $args[1] = '>';
+                    $args[2] = 0;
+                }
+                $args[0] = $model->action('count');
+
+                $this->conditions[] = $args;
```

Comment:
Shouldn't we use `return $this->addCondition(...$args);` here?

Reply:
Yes, can be done. Will switch it over.